app/views/items/index.html.erb

```diff
@@ -25,17 +25,13 @@
 </section>
 <section id="top_five">
   <h2>Top Five Items</h2>
-  <% if @top_five_items %>
-    <% @top_five_items.each do |item| %>
-      <p><%= "#{item.name}: #{item.qty}" %></p>
-    <% end %>
+  <% @items.top_five_items.each do |item| %>
+    <p><%= "#{item.name}: #{item.qty}" %></p>
   <% end %>
 </section>
 <section id="bottom_five">
   <h2>Bottom Five Items</h2>
-  <% if @bottom_five_items %>
-    <% @bottom_five_items.each do |item| %>
-      <p><%= "#{item.name}: #{item.qty}" %></p>
-    <% end %>
+  <% @items.bottom_five_items.each do |item| %>
```

Comment:
Removed the if statement from both top & bottom five, as it's no longer irrelevant. Because we're now calling it on the collection @items instead of directly it will have a value regardless of how this page is accessed.